src/containers/Article.js

```diff
@@ -499,4 +498,4 @@ Article.defaultProps = {
 }
 
 export { Article }
-export default connect(mapStateToProps, { fetchAFullPost })(withLayout(Article))
```

Comment:
If the `app-shell` is going to replace `withLayout` HOC, we might need to check props which belongs to `withLayout` and add them to `app-shell`.
For example : [`theme`](https://github.com/nickhsine/twreporter-react/blob/1d05a1f61a65245447dd8bd9f8b2a95dd5e1ff32/src/helpers/with-layout.js#L86), which determines the background-color.